File: C++/archivos cpp/letter_distance.cpp
#include<iostream>
#include<algorithm>
using namespace std;

int letter_distance(string word1, string word2){
	int distance = 0;
	int length_min = min(word1.length(), word2.length());
	for(int i=0; i<length_min; i++){
		distance += abs((int) word1[i] -  (int) word2[i]); 
	}
	distance += word1.length() - word2.length();
	return distance;
}	

int main(int argc, char const *argv[]){
	cout<<letter_distance("house", "fly");
	return 0;
}

File: Python/Práctica/letter_distance.py

def letter_distance(word1, word2):
	distance = 0
	length_min = min(len(word1), len(word2))
	for i in range(length_min):
		distance += abs(ord(word1[i]) - ord(word2[i]))
	distance+=len(word1) - len(word2)
	return distance

print(letter_distance("abcde", "abcdef"))



